index.html

```diff
@@ -0,0 +1,357 @@
+<!DOCTYPE html>
+<html lang="nl">
+<head>
+    <meta charset="UTF-8">
+    <meta content="IE=edge" http-equiv="X-UA-Compatible">
+    <meta content="width=device-width, initial-scale=1.0" name="viewport">
+    <meta content="detailpagina funda" name="description">
+
+    <!-- Link to CSS -->
+    <link href="styles/style.css" rel="stylesheet" type="text/css">
+
+    <!-- Link to boxicons -->
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+
+
+    <!-- Link to JavaScript script -->
+
+    <script defer src="scripts/v2.js">
+
+    </script>
+    <!--geen defer vanwege error in w3c validator-->
```

Comment:
You mentioned that you can't add `defer`, because of an error with the "w3c validator". However, you added the `defer`. Why did you choose to go for a parallel loading approach for the JavaScript?

Seeing the error in w3c validator can be tricky, because it's not always a reliable external tool for errors in your app. If you're not experiencing errors with your app and the script does what it should, then you could also ignore this. If you do get an error in your app or you want to know more, feel free to ask us.

```suggestion
    <script defer src="scripts/v2.js">

    </script>
    <!--geen defer vanwege error in w3c validator-->
```